addons/test_discuss_full/tests/test_performance.py

```diff
@@ -1052,3 +1052,24 @@ def _get_query_count(self):
             The point of having a separate getter is to allow it to be overriden.
         """
         return 75
+
+    def test_activities_for_lead_lost_opportunity(self):
```

Comment:
You should probably move this test in ``test_mail/test/test_mail_activity.py``; it seems we have few test for systrya in 16.3 so you could probably create a new test class 'TestqactivitySystray', stuff like that.